docs/pulsar_connection.md

```diff
@@ -14,25 +14,98 @@ spec:
   adminServiceURL: http://test-pulsar-sn-platform-broker.test.svc.cluster.local:8080
   brokerServiceURL: pulsar://test-pulsar-sn-platform-broker.test.svc.cluster.local:6650
   clusterName: pulsar-cluster
-  # optional
-  authentication:
+```
+
+Other `PulsarConnection` configuration examples:
+
+* TLS connection
+  
+  ```yaml
+  apiVersion: resource.streamnative.io/v1alpha1
+  kind: PulsarConnection
+  metadata:
+    name: test-tls-pulsar-connection
+    namespace: test
+  spec:
+    adminServiceSecureURL: https://test-pulsar-sn-platform-broker.test.svc.cluster.local:443
+    brokerServiceSecureURL: pulsar+ssl//test-pulsar-sn-platform-broker.test.svc.cluster.local:6651
+    clusterName: pulsar-cluster
+  ```
+
+* JWT Token Auth with Secret
+  
+  ```yaml
+  apiVersion: resource.streamnative.io/v1alpha1
+  kind: PulsarConnection
+  metadata:
+    name: test-tls-pulsar-connection
+    namespace: test
+  spec:
+    adminServiceURL: http://test-pulsar-sn-platform-broker.test.svc.cluster.local:8080
+  brokerServiceURL: pulsar://test-pulsar-sn-platform-broker.test.svc.cluster.local:6650
+  clusterName: pulsar-cluster
+    authentication:
     token:
       secretRef:
         name: test-pulsar-sn-platform-vault-secret-env-injection
         key: brokerClientAuthenticationParameters
-```
+  ```
+
+* JWT Token Auth with value
+
+  ```yaml
+  apiVersion: resource.streamnative.io/v1alpha1
+  kind: PulsarConnection
+  metadata:
+    name: test-tls-pulsar-connection
+    namespace: test
+  spec:
+    adminServiceURL: http://test-pulsar-sn-platform-broker.test.svc.cluster.local:8080
+  brokerServiceURL: pulsar://test-pulsar-sn-platform-broker.test.svc.cluster.local:6650
+  clusterName: pulsar-cluster
+    authentication:
+    token:
+      # JWT Token value should be base64 encoded to use
+      value: ZXlKaGJHY2lPaUpJVXpJMU5pSjkuZXlKemRXSWlPaUowWlhOMExYVnpaWElpZlEuOU9IZ0U5WlVEZUJUWnM3blNNRUZJdUdORVgxOEZMUjNxdnk4bXF4U3hYdw==
```

Comment:
Out of curiosity: Is it a real value or a sample value?

Reply:
I took the token from https://pulsar.apache.org/docs/3.0.x/security-jwt/ then to have a base64 encode.